FWCore/Framework/src/NumberOfConcurrentIOVs.h

```diff
@@ -38,6 +38,8 @@ namespace edm {
 
       void initialize(ParameterSet const* eventSetupPset);
 
+      void initialize(unsigned int nStreams, unsigned int nConcurrentLumis);
```

Comment:
Having three functions named `initialize` with no documentation as to what they are used for does not seem like a good idea to me.

Reply:
I renamed the functions. That is better, thanks.